scripts/cxg_admin.py

```diff
@@ -138,6 +150,29 @@ def transfer_collections(ctx, curr_owner, new_owner):
                 exit(0)
 
 
+@cli.command()
+@click.pass_context
+def create_cxg_artifacts(ctx):
+    with db_session_manager() as session:
+        datasets = session.query(DbDataset.id, DbDataset.explorer_url).all()
+        session.query(DbDatasetArtifact).filter(DbDatasetArtifact.filetype == DatasetArtifactFileType.CXG).delete()
+        session.commit()
+        for dataset in datasets:
+            if dataset.explorer_url:
+                object_key = dataset.explorer_url.split("/")[-2]
+                s3_uri = f"s3://{cxg_bucket.name}/{object_key}/"
+                print(dataset.explorer_url, s3_uri)
```

Comment:
Intentional logging statement?

Reply:
Yes. It's nice for sanity checking everything was created correctly. 
Once the data migration is run I'll probably remove this script to avoid creating bad data. In the future the explorer url and the s3uri may not contain the same ids (revision work)